react/components/PasswordInput/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import Input from '../Input'
+import VisibilityOn from '../icon/VisibilityOn'
+import VisibilityOff from '../icon/VisibilityOff'
+
+class PasswordInput extends Component {
+  state = {
+    showPassword: false,
+  }
+
+  toggle = () => this.setState(state => ({ showPassword: !state.showPassword }))
+
+  render() {
+    const size = this.props.size
+    const iconSize = size === 'large' ? 18 : size === 'x-large' ? 22 : 16
```

Comment:
I'd rather do it in a more explicit way:

```js
const iconSizes = {
  'default': 16,
  'large': 18,
  'x-large': 22,
}

const iconSize = iconSizes[size] || iconSizes.default
```

Reply:
Sure @felippenardi! But was keeping the same pattern other components use.